README.md

```diff
@@ -65,11 +65,11 @@ Tina Raissi is finishing her Master’s studies in Computer Science and working
 
 ---
 
-### Nixie Sapphira Lesmana (Voice)
+### Nixie Sapphira Lesmana (Vocals)
```

Comment:
could you change all the other performers to "Vocals" as well?

Reply:
Sure